modules/axhal/src/platform/pc_x86/acpi.rs

```diff
@@ -343,12 +319,16 @@ pub(crate) fn init() {
 }
 
 /// Get PCI IRQ and map it to vector used in OS.
+/// Temporarily allow unused here because irq support for virtio hasn't ready yet.
+#[allow(dead_code)]
+#[cfg(feature = "irq")]
 pub fn get_pci_irq_vector(bus: u8, device: u8, function: u8) -> Option<usize> {
     unsafe { ACPI.get_pci_irq_desc(bus, device, function) }
         .map(|irq_desc| irq_to_vector(irq_desc.irq as u8))
 }
 
 /// Get PCIe ECAM space physical address.
+#[allow(dead_code)]
 pub fn get_ecam_address() -> Option<u64> {
```

Comment:
Call this function in `axdriver` to get the PCI ECAM address, then we can remove the hardcoding in `axconfig`.